modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
+
+Copy the command to access a running virtual machine (VM) via SSH.
+
+.Procedure
+
+. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. Use the search and filters to find your VM in the list.
```

Comment:
I think step 2 should be "Select a virtual machine to open the **VirtualMachine details** page.

Reply:
@sjhala-ccs This step is prior to opening the VM details page. It's a kebab accessed from the list of VMs.

Reply:
@sabrinajess If it's the kebab menu for the specific VM, then you should edit the next step to say:

"Click the Options menu {kebab} for your virtual machine and select *Copy SSH command*. You can now paste this command onto the OpenShift CLI (`oc`)."

The {kebab} variable renders as the icon in the published doc.